src/main/java/org/fogbowcloud/manager/core/models/exceptions/RequestException.java

```diff
@@ -0,0 +1,12 @@
+package org.fogbowcloud.manager.core.models.exceptions;
+
+import org.fogbowcloud.manager.core.models.ErrorType;
+
+public class RequestException extends Exception {
```

Comment:
Remember to sincronize with the team about this exception. What is the context that its is used ?

Reply:
I'm using just to substitute OCCI exception. However we need to create an exception hierarchy to cover all kinds of exceptions.